### PR TITLE
fix: add skipLibCheck and remove wrap-ansi resolution

### PR DESCRIPTION
**Pull request by schultzp2020**
## Description

Please explain the changes you made here.

## Which issue(s) does this PR fix

- Fixes #?

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -25643,7 +25643,16 @@ wordwrap@^1.0.0:
...
+wrap-ansi@^6.0.1, wrap-ansi@^6.2.0:
+  version "6.2.0"
+  resolved "https://registry.yarnpkg.com/wrap-ansi/-/wrap-ansi-6.2.0.tgz#e9393ba07102e6c91a3b221478f0257cd2856e53"
+  integrity sha512-r6lPcBGxZXlIcymEu7InxDMhdW0KDxpLgoFLcguasxCaJ/SOIZwINatK9KY/tf+ZrlywOKU0UDj3ATXUBfxJXA==
+  dependencies:
+    ansi-styles "^4.0.0"
+    string-width "^4.1.0"
+    strip-ansi "^6.0.0"
+
+wrap-ansi@^7.0.0:
```
Comment (nickboldt) [inline yarn.lock:25655]: if you're not pinning to 7, can we remove it entirely and just use 8.0 or 8.1?

**Review by nickboldt — APPROVED** [review]
seems ok, with one question

File: yarn.lock (hunk 1 above)
Comment (schultzp2020) [reply yarn.lock:25655]: Sadly we cannot because v8 only supports ESM.

Comment (nickboldt) [reply yarn.lock:25655]: do why do you need to include 6.2,7, and 8.1 in the yarn.lock?

**Pull request merged**